packages/integration-tests/src/test-project-setup/test_project_base.ts

```diff
@@ -183,6 +183,12 @@ export abstract class TestProjectBase {
       await currentCodebaseBackendOutputClient.getOutput(backendId);
     const npmOutputs = await npmBackendOutputClient.getOutput(backendId);
 
+    console.log(
+      '🪵 currentCodebaseOutputs: ',
+      JSON.stringify(currentCodebaseOutputs, null, 2)
+    );
+    console.log('🪵 npmOutputs: ', JSON.stringify(npmOutputs, null, 2));
+
```

Comment:
adding the log to make it easier to compare if the output updates are intented. Otherwise, it's hard to see what the differences are. e.g. https://github.com/aws-amplify/amplify-backend/actions/runs/10120959378/job/28002086625

Reply:
These may include API_KEY.
We should not log them.

We may want to introduce a flag to have this locally, but we shouldn't do this in CI.